examples/hotrod/pkg/tracing/rpcmetrics/observer_test.go

```diff
@@ -106,39 +104,20 @@ func TestObserver(t *testing.T) {
 
 func TestTags(t *testing.T) {
 	type tagTestCase struct {
-		key     string
-		variant string
-		value   interface{}
+		attr    attribute.KeyValue
 		metrics []u.ExpectedMetric
 	}
-
-	testCases := []tagTestCase{
-		{key: "something", value: 42, metrics: []u.ExpectedMetric{
-			{Name: "requests", Value: 1, Tags: tags("error", "false")},
-		}},
-		{key: "error", value: true, metrics: []u.ExpectedMetric{
-			{Name: "requests", Value: 1, Tags: tags("error", "true")},
-		}},
-		// OTEL bridge does not interpret string "true" as error status
-		// {key: "error", value: "true", variant: "string", metrics: []u.ExpectedMetric{
-		// 	{Name: "requests", Value: 1, Tags: tags("error", "true")},
-		// }},
-	}
+	testCases := []tagTestCase{}
```

Comment:
I don't understand your change - why are you deleting test cases? They had distinct outcomes in the expected metrics.

Reply:
sorry! I thought they are getting ignored against the metrics created during spans starting since error=true has no significance for OTEL.

Reply:
error cases have the same significance in OTEL, they are just expressed differently, via status instead of a tag

Reply:
so how I can re declare it as status? any OTEL fmt?

Reply:
didn't my comment answer that yesterday? https://github.com/jaegertracing/jaeger/pull/4635/files#r1290509959